app/controllers/api/mixins/service_templates.rb

```diff
@@ -24,8 +24,9 @@ def api_request_allowed?
       end
 
       def request_from_ui?
+        $api_log.info("requester_type: #{params['requester_type']}")
```

Comment:
I only had this for testing, probably doesn't actually need to be here.

Reply:
It's not working anyway :( The param's being stripped out somewhere.